app/_locales/en/messages.json

```diff
@@ -1887,6 +1887,16 @@
   "experimental": {
     "message": "Experimental"
   },
+  "experimentalBitcoinSectionTitle": {
```

Comment:
Also wonder if we should name it differently like `bitcoinSupportSectionTitle`? Or if you want to keep the "experimental" prefix, then maybe `experimentalBitcoinSupportSectionTitle" (so that we still have "BitcoinSupport" as part of the name to match with the flag name

Reply:
[20e356b](https://github.com/MetaMask/metamask-extension/pull/25672/commits/20e356b53e25c0ff6f4a6dbc965042e052d92654)